<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectWindows</class>
 <widget class="QMainWindow" name="DetectWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1011</width>
      <height>731</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::TabPosition::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::TabShape::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_MainMonitor">
     <attribute name="title">
      <string>Main Monitor</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_Screen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>821</width>
        <height>631</height>
       </rect>
      </property>
      <property name="title">
       <string>Screen</string>
      </property>
      <widget class="QFrame" name="frame_Detection">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>800</width>
         <height>600</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_Production_model">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>971</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Production Model</string>
      </property>
      <widget class="QLabel" name="label_Model_Name">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>551</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="text">
        <string>Model Name</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::PlainText</enum>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>20</y>
         <width>16</width>
         <height>41</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_Classes">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>20</y>
         <width>371</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="text">
        <string>Lead / Lead Free</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>180</y>
        <width>541</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Model Name</string>
      </property>
      <property name="textFormat">
       <enum>Qt::TextFormat::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_Setting">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>70</y>
        <width>141</width>
        <height>531</height>
       </rect>
      </property>
      <property name="title">
       <string>Setting</string>
      </property>
      <widget class="QCheckBox" name="checkBox_signal_Alarm">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Signal Alarm</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_SoundAlarm">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>380</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Sound Alarm</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_DigitalOut">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>410</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Digital Out</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_DigitalIN">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>440</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Digital In</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Blank3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>470</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>blank #3</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_blank4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>500</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>blank #4</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_confident">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>92</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <property name="text">
        <string>Confident =</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_confident">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::ButtonSymbols::PlusMinus</enum>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>70</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_connect">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_disconnect">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>54</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_confident">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>70</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber_CheckOK">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_CheckOK">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <property name="text">
        <string>Check OK Qty =</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_CheckNG">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <property name="text">
        <string>Check NG Qty =</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber_CheckNG">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::SegmentStyle::Filled</enum>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_Logo">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>600</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Universal_AI2/agc-logo-png-transparent-logo-v2.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Model_Matching">
     <attribute name="title">
      <string>Model Matching</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_Model_match_making">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>981</width>
        <height>651</height>
       </rect>
      </property>
      <property name="title">
       <string>Model Match Making</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QGroupBox" name="groupBox_Browse_Yolo">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>20</y>
         <width>351</width>
         <height>511</height>
        </rect>
       </property>
       <property name="title">
        <string>Browse Yolo Model</string>
       </property>
       <widget class="QLabel" name="label_yolo_model">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>27</y>
          <width>221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Panel</enum>
        </property>
        <property name="text">
         <string>Select Yolo Model (.pt)</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QDialogButtonBox" name="buttonBox_yolo_open_apply">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>20</y>
          <width>81</width>
          <height>61</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Open</set>
        </property>
        <property name="centerButtons">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>321</width>
          <height>411</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_Model">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>571</width>
         <height>621</height>
        </rect>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
       <widget class="QTableWidget" name="tableWidget_model_match">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>551</width>
          <height>551</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::PenStyle::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Classes</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Status</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Barcode</string>
         </property>
        </column>
       </widget>
       <widget class="QDialogButtonBox" name="buttonBox_save_cancel_apply">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>570</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Save</set>
        </property>
        <property name="centerButtons">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_add_model">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>577</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_remove_model">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>577</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_DigitalOutput">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>540</y>
         <width>351</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>Digital Output</string>
       </property>
       <widget class="QLabel" name="label_OK_output">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="text">
         <string>OK Output</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_NG_output">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="text">
         <string>NG Output</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_OK_output_script">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>20</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_NG_output_script">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>60</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>650</y>
        <width>71</width>
        <height>46</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Universal_AI2/agc-logo-png-transparent-logo-v2.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
